Gems/WarehouseAutomation/Assets/Factory/ConveyorElements/ConveyorSegments_Turnout.prefab

```diff
@@ -0,0 +1,2218 @@
+{
```

Comment:
Please adjust length of spline.
![Screenshot from 2023-08-29 18-25-01](https://github.com/o3de/o3de-extras/assets/3209244/40977238-c169-4d50-bb1c-1d5fd2144fc2)

Reply:
Good catch!